wee_slack.py

```diff
@@ -887,7 +887,7 @@ def nick_completion_cb(data, completion_item, current_buffer, completion):
     current_buffer = w.current_buffer()
     current_channel = EVENTROUTER.weechat_controller.buffers.get(current_buffer, None)
 
-    if current_channel is None or current_channel.members is None:
+    if current_channel is None or getattr(current_channel, "members", None) is None:
```

Comment:
It would be nice to have the `members` property on `SlackTeam` as well, but we don't have a list of all members on the team, so it would be misleading. So maybe best to just have it like this.

Reply:
`members` could be constructed from `users` (e.g. `self.members = [user.identifier for user in self.users]`), but I'm not sure how useful nick completion would be in the core buffer anyway.

Reply:
Ah, forgot about that. Having users in the team buffer is nice for opening a query with someone. Also, it removes the need to check if the attribute is present.

Though, we shouldn't copy the list, as we would have to keep the lists in sync. But you can add this in `SlackTeam`:

```
    @property
    def members(self):
        return self.users
```

Reply:
Even better!